.github/workflows/schema-validator.yml

```diff
@@ -16,14 +16,14 @@ jobs:
           file: "tokens/movistar.json"
           schema: "tokens/schema/skin-schema.json"
 
-  Movistar-classic:
+  Movistar-legacy:
```

Comment:
why this change is appearing when this already up in pre-production? :S
https://github.com/Telefonica/mistica-design/blob/a2fd5e24cc7f1f80ef32e64d8f62bf0764b29098/.github/workflows/schema-validator.yml#L19

Reply:
Pre-production should be merged into this branch first, then these changes that are already in pre will dissapear

Reply:
this is ready @yceballost